pkg/storageincentives/redistribution/redistribution.go

```diff
@@ -209,3 +208,12 @@ func (c *contract) callTx(ctx context.Context, callData []byte) ([]byte, error)
 	}
 	return result, nil
 }
+
+func (c *contract) Fee(ctx context.Context, txHash common.Hash) *big.Int {
```

Comment:
method `Fee` seems to not be responsability of this structure

Reply:
The fee is calculated when we perform transactions on `claim`, `reveal`, and `commit` functions. So we require a cumulative fee for these functions. That is why I have placed it here.

Reply:
sure, but this method is not depended on `contract` therefore it shouldn't be defined here. would it make sense to move this method to `RedistributionState` structure?

Reply:
This method is already being called in `RedistributionState` structure `Fee` method.

Reply:
If no one other then `RedistributionState` is calling this `Fee` method, then it doesn't make much sense to have this method defined here instead of having it defined in `RedistributionState`. This would reduce (remove it entirely) dependency between `RedistributionState` and `contract`.